### PR TITLE
 iOS build in Release mode failes with MTOUCH MT5210 errors

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] ~Did you add unit tests if this modifies the Windows code?~
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Related PRs or issues

[AB#76735](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/76735)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ App Center Data is [retired](https://aka.ms/MBaaS-retirement-blog-post) and has
 * **[Fix]** Allow whitespace characters between secrets like in versions before `2.6.0`.
 
 #### iOS
+* **[Fix]** Fix reference to Security framework which can be missed in release builds with Xamarin.iOS 13.10 or newer.
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion
* **[Fix]** Fix reference to `Security` framework which can be missed in release builds with Xamarin.iOS 13.10 or newer.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ App Center Data is [retired](https://aka.ms/MBaaS-retirement-blog-post) and has
 * **[Fix]** Allow whitespace characters between secrets like in versions before `2.6.0`.
 
 #### iOS
+* **[Fix]** Fix reference to `Security` framework which can be missing in release builds with Xamarin.iOS 13.10 or newer.
```
Comment (guperrot) [inline CHANGELOG.md]:
```suggestion

* **[Fix]** Fix reference to `Security` framework which can be missing in release builds with Xamarin.iOS 13.10 or newer.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,8 @@ App Center Data is [retired](https://aka.ms/MBaaS-retirement-blog-post) and has
 * **[Fix]** Allow whitespace characters between secrets like in versions before `2.6.0`.
 
 #### iOS
+
+* **[Fix]** Fix reference to `Security` framework which can be missing in release builds with Xamarin.iOS 13.10 or newer.
```
Comment (guperrot) [inline CHANGELOG.md]: Let's move to 3.0.1 next release thus targeting develop after release.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,14 @@
 # App Center SDK for .NET Change Log
 
+## Version 3.0.1 （Under development）
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion
## Version 3.0.1 (Under development)
```

**Pull request merged**